packages/babel-core/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "7.7.0",
   "description": "Babel compiler core.",
   "main": "lib/index.js",
+  "_main:used-by-babel-standalone": "src/index.js",
```

Comment:
By prefixing a pragmatic underscore and suffixing a paranoid signature I hope none of our user will depend on these fields in their bundling scripts. It should not kick in for most of users who are bundling _with_ babel instead of bundling babel itself.